data.sql

```diff
@@ -1,5 +1,14 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
+  VALUES ('Agumon', '02/03/2020', 0, true, 10.23);
+
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
+  VALUES ('Gabumon', '11/15/2018', 2, true, 8);
+
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
+  VALUES ('Pikachu', '06/07/2021', 1, false, 15.04);
```

Comment:
- [ ] Kindly make sure to use the correct value of `date_of_birth`, it should be `'2021/01/07'`

Reply:
Required change applied with
https://github.com/luigirazum/pawclinic-db/pull/1/commits/27e2e12567791b5f6a8dfd16ba2be14416b1892c